File: workspace/pset6/Sentimental/cash/cash.py
from cs50 import get_float
import math

def main():
    userinput = 0.0

    userinput = get_float("Input: ")
    while(userinput <= 0.0):
        userinput = get_int("Input: ")


    change = round(userinput * 100)
    print(f"Change = {change} cents")

    numQuarters = 0
    numDimes = 0
    numNickels = 0
    numPennies = 0
    if change / 25 > 0:
        numQuarters = math.floor(change / 25)
        change = change % 25
    if change / 10 > 0:
        numDimes = math.floor(change / 10)
        change = change % 10
    if change / 5 > 0:
        numNickels = math.floor(change / 5)
        change = change % 5

    numPennies = change

    val =  numQuarters + numDimes + numNickels + numPennies
    print(f"{val}")

if __name__ == "__main__":
    main()

File: workspace/pset1/cash/cash.c
#include <stdio.h>
#include <cs50.h>
#include <math.h>

int main(void) {
    float input = 0.0;

    do {
        input = get_float("Change owed: ");
    } while(input < 0);

    int change = round(input * 100);
    printf("Change = %i\n", change);

    int numQuarters = 0;
    int numDimes = 0;
    int numNickels = 0;
    int numPennies = 0;
    if(change / 25 > 0){
        numQuarters = change / 25;
        change = change % 25;
    }
    if(change / 10 > 0){
        numDimes = change / 10;
        change = change % 10;
    }
    if(change / 5 > 0){
        numNickels = change / 5;
        change = change % 5;
    }
    numPennies = change;

    printf("%i\n", numQuarters + numDimes + numNickels + numPennies);
}